### PR TITLE
Designated Queues per Resource Capability

### PR DESCRIPTION
**Pull request by JVickery-TBS**
feat(dev): designated queues;

- Add capability for designated queues per resource.

### CODE REVIEW

File: .github/workflows/change_log.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            echo -e "\n"
+            echo -e "\033[0;36mINFO: extension ${extension} accepted.\033[0;0m"
+            echo -e "\n"
+            exit 0
+          else
+            echo -e "\n"
+            echo -e "\033[0;31mERROR: changelog file ending in ${extension} not supported.\033[0;0m"
+            echo -e "\n"
+            exit 1
+          fi
```
Comment (wardi) [inline .github/workflows/change_log.yml:43]: unrelated change?

**Comment by wardi** [comment]
same question about the new option: how does the behviour change? Best to document the option in the readme and explain there.

File: .github/workflows/change_log.yml (hunk 1 above)
Comment (JVickery-TBS) [reply .github/workflows/change_log.yml:43]: Yeah, our ckanext-validation fork just did not have the changelog workflow yet for our new Release Notes thing

**Comment by JVickery-TBS** [comment]
https://github.com/open-data/ckanext-xloader/pull/28#issuecomment-2130268631

**Comment by JVickery-TBS** [comment]
@wardi I have added a new section to the README for our fork, describing some of the main changes of our fork. This includes the new capability for the dedicated queues.

**Review by wardi — APPROVED** [review]
